packages/web/src/js/dom/event-handler.js

```diff
@@ -0,0 +1,15 @@
+const addHandler = (element, eventType, handler) => element.addEventListener(eventType, handler);
+
+const removeHandler = (element, eventType, handler) => element.removeEventListener(eventType, handler);
+
+const EventHandler = {
+  on(element, event, handler) {
+    addHandler(element, event, handler);
+  },
+
+  off(element, event, handler) {
+    removeHandler(element, event, handler);
+  },
+};
+
+export default EventHandler;
```

Comment:
You are right. I've chosen a more general approach to preparing Spirit scripts. I want to encapsulate a few things to have one place where I can manage and modify them. Although this encapsulation may look like that adds nothing to it, maybe in the future it could save us some time (have a look on a bootstrap - https://github.com/twbs/bootstrap/blob/main/js/src/dom/event-handler.js :-))

Reply:
OK